pytest_monitor/pytest_monitor.py

```diff
@@ -154,6 +157,8 @@ def prof():
     if not PYTEST_MONITORING_ENABLED:
         wrapped_function()
     else:
+        if pyfuncitem.session.config.option.mtr_no_gc:
```

Comment:
would it make the code more readable if the flag is called 'mtr_enable_gc'? otherwise it reads as if shouldn't be run if `mtr_no_gc` is True.

Reply:
You are absolutely right, I forgot a not in the condition. I should have been taught not to write code at undue hours...
👍🏻 Good point for the option container.

Reply:
I personally prefer normal variables and turning them off with `--gc 0` and default `1`/`True` - otherwise things aren't as straightforward, especially double negations can be tricky.